package.json

```diff
@@ -25,7 +25,7 @@
     "link-all": "node scripts/link-all.js packages",
     "unlink-all": "node scripts/unlink-all.js packages",
     "check": "flow check",
-    "lint": "eslint . && prettier \"./packages/*/*/{src,bin,test}/**/*.{js,json,md}\" --list-different && cargo fmt --all -- --check",
+    "lint": "eslint . && prettier \"./packages/*/*/{src,bin,test}/**/*.{js,json,md}\" --list-different && cargo +nightly fmt --all -- --check",
```

Comment:
Why is `+nightly` needed? We install Rust stable in CI so this fails to work.

Reply:
Using nightly formatting features (basically all of the fmt options are nightly - something about not wanting to make formatting stable). This doesn't affect the compiler, only using nightly for the formater. 

The command fails when it gets to `cargo fmt` but CI doesn't fail (I guess we don't have `set -e` in the script?).

Reply:
Yeah, sorry, that's what I meant - I think you'll need to update CI to also install the `nightly` tooling (since currently it only installs the `stable` tooling)..

Reply:
We should remove the nightly formatting options so we can avoid having to install it.

It's not worth the hassle, there's no benefit from those linter options and they are not standards across the community.

Having to install two toolchains is a downside.